src/menu.ts

```diff
@@ -742,10 +743,6 @@ export default class Menu extends LitElement {
     }
   }
 
-  #onDefaultSlotMouseUp() {
-    this.#isDefaultSlotClick = true;
```

Comment:
Menu can set this in `#onDefaultSlotClick()` now that `#onDocumentClick()` [no longer](https://github.com/CrowdStrike/glide-core/pull/986/files#diff-5fe1f96ff85f15ff3232204c90e15722ca02da3ebf73ec21cf69b7abe2ac25ac) handles clicks in their capture phase.